models/Discord/Commands/DiscordCommandInfo.js

```diff
@@ -23,6 +23,14 @@ class DiscordCommandInfo extends DiscordCommand {
 				if ('error' in results) {
 					return message.reply(results.error);
 				}
+				var colour = "";
+				var colours = ['0', '1', '2', '3', '4', '5', '6', '7', '8', '9', 'a', 'b', 'c', 'd', 'e', 'f'];
+				var count = 0;
+
+				for(count = 0; count < 6; count++) {
```

Comment:
`for(var count = 0; count < 6; count++) `

And remove the above count variable